### ISSUE TITLE
Value 0 throws error

### ISSUE THREAD

**Issue by EdLipson5117** [TheBlupper/winv#1]
winv.set_volume(int(vol)) with vol = 0 throws:

C:\Users\edlip\AppData\Local\Programs\Python\py9aQt\lib\site-packages\winv\__init__.py:42: UserWarning: -28.126100540161133 is not between -28 and 0 (the system volume scale) and therefore it is possible for the output to be outsideof the 0-100 range
  warnings.warn(f'{x} is not between -28 and 0 (the system volume scale) and therefore it is possible for the output to be outsideof the 0-100 range')

Python 3.9.1

**Comment by TheBlupper** [comment]
This module has broken since a couple of windows updates ago, and I do not have any plans to reopen the code. This reminded me that I sohuld probably take it down from PyPi to avoid further confusion, sorry about that. I did also find that it did behave unpredictably on other systems, so it is probably for the better. I have not done too much research since then, but [this library](https://pypi.org/project/winsystem/) and others have popped up. Maybe give it a try. Closing this thread since the project is no longer worked on.
